modules/ref_ipi-installation-overview.adoc

```diff
@@ -0,0 +1,48 @@
+
+:_mod-docs-content-type: REFERENCE
+
+[id="installation-overview_{context}"]
+= Installation overview
+
+The installation program supports interactive mode, but Red{nbsp}Hat recommends preparing an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts, and the relevant cluster details.
+
+The installation program loads the `install-config.yaml` file, while the administrator generates the manifests and verifies all prerequisites.
+
+The installation program enrolls all nodes in the cluster and starts the bootstrap VM. The installation program starts the metal platform components as `systemd` services, which have the following containers:
+
+* Ironic-dnsmasq: The DHCP server responsible for handing over the IP addresses to the provisioning interface of various nodes on the provisioning network. Ironic-dnsmasq is only enabled when you deploy an {product-title} cluster with a provisioning network.
+* Ironic-httpd: The HTTP server used to ship the images to the nodes.
+* Image-customization
+* Ironic
+* Ironic-inspector (available in {product-title} 4.16 and earlier)
+* Ironic-ramdisk-logs
+* Extract-machine-os
+* Provisioning-interface: The service that configures networking when the provisioning interface is enabled.
+* Metal3-baremetal-operator: The operator that provides a higher-level API for Ironic and replaces `terraform-provider-ironic` (available in {product-title} 4.16 and later)
+
+The nodes enter the validation phase, where each node moves to a “manageable” state after Ironic validates the credentials to access the Baseboard Management Controller (BMC).
+
+Once in the “manageable” state, the “inspection” phase starts. The "inspection" phase ensures that the hardware meets the minimum requirements needed for a successful deployment of {product-title}.
+
+The `install-config.yaml` file details the provisioning network. The installation program on the bootstrap VM will try to push a live image by using PXE to every node with the Ironic Python Agent (IPA) loaded. When using virtual media, it connects directly to each node's BMC to virtually attach the image.
+
+When using PXE boot, all nodes reboot to start the process:
+
+* The `ironic-dnsmasq` service running on the bootstrap VM provides the IP address of the node and the TFTP boot server.
+* The first-boot software loads the root file system over HTTP.
+* The `ironic` service on the bootstrap VM receives the hardware information from each node.
+
+The nodes then enter the “cleaning” state, where each node must clean all the disks before continuing with the configuration.
+
+Once the “cleaning” state finishes, the nodes enter the “available” state and the installation program moves the nodes to the “deploying” state.
+
+IPA runs the `coreos-installer` to install the {op-system-first} image on the disk defined by `rootDeviceHints` in the `install-config.yaml` file. The node boots by using {op-system}.
```

Comment:
_the `coreos-installer`_ requires a noun. In a quick search, _command_, _utility_, _program_ all see common in the docs. Similar with `rootDeviceHints`, as it is in backticks.
```suggestion
IPA runs the `coreos-installer` command to install the {op-system-first} image on the disk defined by the `rootDeviceHints` parameter in the `install-config.yaml` file. The node boots by using {op-system}.
```

Reply:
Updated as suggested.